scripts/aggregate-translations/aggregate-translations.js

```diff
@@ -36,7 +36,7 @@ const defaults = (options = {}) => {
     locales: options.locales || config.locales || supportedLocales,
     outputDir: options.outputDir || './aggregated-translations',
     excludes: options.excludes || config.excludes || [],
-    format: options.format,
+    format: options.format || 'es5',
```

Comment:
Can we update
```diff
- fileSystem: options.outputFileSystem || config.outputFileSystem || fse,
+ fileSystem: config.outputFileSystem || fse,
```

if you look at how this option is used the aggregate-translation, this could not be provided by the CLI tool (and is not an option) because it would need to be passed as module.

Reply:
I'd leave it as-is because the `options` is not specific to CLI. The user could still use the [setup function](https://github.com/cerner/terra-toolkit/blob/master/docs/AggregateTranslations.md#setup-example) directly and pass the `outputFileSystem`, me thinks.

Reply:
Ah I see what you mean. 👍